projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad.php

```diff
@@ -354,6 +354,21 @@ function wpcom_register_default_launchpad_checklists() {
 		)
 	);
 
+	wpcom_register_launchpad_task_list(
+		array(
+			'id'       => 'design-first',
+			'title'    => 'Pick a Design',
+			'task_ids' => array(
+				'design_selected',
+				'setup_blog',
+				'domain_upsell',
+				'plan_completed',
+				'first_post_published',
```

Comment:
As we talked, we're going to hide this, for the first iteration, [directly on Calypso](https://github.com/Automattic/wp-calypso/pull/77222/commits/d10c4f2284738e34e9650f366b901a824862498c).